src/main/java/org/team1540/base/power/PowerManager.java

```diff
@@ -127,47 +138,35 @@ private double scaleExponential(double highestPriority, double priority) {
     A comment on the following two methods: Why are they synchronized? Why not use a more thread-safe data structure
 	that better supports the intended use case? Because it doesn't really matter.
 
-	In theory, it'd be nice to be able to say "Hey, this command isn't being used any more, let's ignore it when we're
+	In theory, it'd be nice to be able to say "Hey, this PowerManageable isn't being used any more, let's ignore it when we're
 	doing our calculations" while we're actually doing those calculations. But that actually isn't really a big deal.
-	Any time we actually do need to register/unregister a command, it'll happen immediately after we're done managing
+	Any time we actually do need to register/unregister a PowerManageable, it'll happen immediately after we're done managing
 	the power for that cycle. Furthermore, if we allocate it power it doesn't use, that's not a huge problem for
 	everyone else. There's also no good reason to access the map outside of the methods defined here, so just don't do
 	it.
 	 */
 
   /**
-   * Registers the command's subsystems as being used. Blocks power scaling.
+   * Registers the {@link PowerManageable} as being used. Blocks power scaling.
    *
-   * @param toRegister The command to register.
+   * @param toRegister The {@link PowerManageable} to register.
    */
-  synchronized void registerCommand(ChickenCommand toRegister) {
-    // For every subsystem the command uses, add it to runningSubsystems; if it's already in there, increment the
-    // priority by the priority of the given command
-    for (ChickenSubsystem currentSubsystem : toRegister.getUsedSubsystems()) {
-      Double currentValue = runningSubsystems.get(currentSubsystem);
-      if (currentValue == null) {
-        currentValue = 0d;
-      }
-      runningSubsystems.put(currentSubsystem, currentValue + toRegister.getPriority());
+  synchronized void registerPowerManageable(PowerManageable toRegister)
+      throws AlreadyBoundException {
+    // For every PowerManageable, add it to powerManaged; if it's already there, throw an exception
+    if (powerManaged.contains(toRegister)) {
+      throw new AlreadyBoundException("PowerManageable " + toRegister + " is already registered.");
```

Comment:
Does it really need to throw an exception? Just have the method return a boolean about if it actually got added.